src/main/java/uk/gov/hmcts/reform/jobscheduler/controllers/JobsController.java

```diff
@@ -38,7 +42,7 @@ public JobsController(JobsService jobsService, AuthService authService) {
     @PostMapping(path = "")
     @ApiOperation("Create a new job")
     public ResponseEntity<Void> create(
-        @RequestBody Job job,
+        @Valid @RequestBody Job job,
```

Comment:
I know that technically it is honky dory, but having annotations in separate lines would be very appreciated. 🥇

Reply:
Sorry, I just noticed it's in method... Ignore me!